packages/Php70/src/Dto/VariableAssignPair.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php70\Dto;
```

Comment:
We have `ValueObject` namespace convention for any non-services object, to keep structure simple